src/main/java/ca/corefacility/bioinformatics/irida/ria/security/CredentialsExpiredAuthenticationFailureHandler.java

```diff
@@ -33,16 +33,17 @@
  *
  */
 @Component
-public class CredentialsExpriredAuthenticationFailureHandler extends SimpleUrlAuthenticationFailureHandler {
-	private static final Logger logger = LoggerFactory.getLogger(CredentialsExpriredAuthenticationFailureHandler.class);
+public class CredentialsExpiredAuthenticationFailureHandler extends SimpleUrlAuthenticationFailureHandler {
+	private static final Logger logger = LoggerFactory.getLogger(CredentialsExpiredAuthenticationFailureHandler.class);
 
 	private final PasswordResetService resetService;
 	private final UserService userService;
 
 	@Autowired
-	public CredentialsExpriredAuthenticationFailureHandler(PasswordResetService resetService, UserService userService) {
+	public CredentialsExpiredAuthenticationFailureHandler(PasswordResetService resetService, UserService userService) {
 		this.resetService = resetService;
 		this.userService = userService;
+		this.setUseForward(true);
```

Comment:
If redirects want to be kept, then we could change this to `this.getRedirectStrategy().setContextRelative(true)`